doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -95,6 +95,20 @@ nRF Desktop
     See the documenation page of nRF Desktop's :ref:`nrf_desktop_hid_forward` for details.
   * Fixed an issue that was causing the HID keyboard LEDs to remain turned on after host disconnection while no other hosts were connected.
 
+Bluetooth® LE
+-------------
+
+* Updated:
+
+  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
```

Comment:
* Updated ~~some~~ samples with support for...

Reply:
That "some" was actually added by me. From what I could tell, not all Bluetooth LE samples were updated.

Reply:
Well... it is just a comment. I really do not like "some" in a documentation and the rest of the comment explains with samples configuration was added. I am not blocking merge anyway - As I marked in my approval - if documentation team agrees that I have nothing to add.